sdk/search/test-resources.json

```diff
@@ -62,7 +62,7 @@
       "properties": {
         "replicaCount": 1,
         "partitionCount": 1,
-        "hostingMode": "default",
+        "hostingMode": "Default",
```

Comment:
This should be `default` based on the documentation I've read, but it also doesn't apply to the SKU being used. So, not sure if this could cause any issues.